backend/src/main/java/pl/zzpj/esportbetting/rest/AuthRestController.java

```diff
@@ -18,12 +18,12 @@
 
 @RestController
 @RequestMapping(path = "/auth")
-public class AuthController {
+public class AuthRestController {
```

Comment:
Jak już zmieniamy nazwy, to proponuję rozwinąć do ```AuthenticationRestController```, że by się nie myliło ewentualnie z ```Authority```